modules/wmi/charts.go

```diff
@@ -2175,6 +2234,512 @@ var (
 	}
 )
 
+// .NET
+var (
+	netFrameworkCLRCharts = module.Charts{
+		netFrameworkCLRExceptionsThrown.Copy(),
+		netFrameworkCLRExceptionsFilters.Copy(),
+		netFrameworkCLRExceptionsFinallys.Copy(),
+		netFrameworkCLRExceptionsThrowCatchDepth.Copy(),
+		netFrameworkCLRInteropCOMCallableWrapper.Copy(),
+		netFrameworkCLRInteropMarshalling.Copy(),
+		netFrameworkCLRInteropStubsCreated.Copy(),
+		netFrameworkCLRJITMethods.Copy(),
+		netFrameworkCLRJITTime.Copy(),
+		netFrameworkCLRJITStandardFailure.Copy(),
+		netFrameworkCLRJITILByes.Copy(),
+		netFrameworkCLRLoadingLoaderHeapSize.Copy(),
+		netFrameworkCLRLoadingAppDomainsLoaded.Copy(),
+		netFrameworkCLRLoadingAppDomainsUnloaded.Copy(),
+		netFrameworkCLRLoadingAssembliesLoaded.Copy(),
+		netFrameworkCLRLoadingClassesLoaded.Copy(),
+		netFrameworkCLRLoadingClassLoadFailure.Copy(),
+		netFrameworkCLRLockAndThreadsQueueLength.Copy(),
+		netFrameworkCLRLockAndThreadsCurrentLogicalThreads.Copy(),
+		netFrameworkCLRLockAndThreadsCurrentPhysicalThreads.Copy(),
+		netFrameworkCLRLockAndThreadsRecognizedThreads.Copy(),
+		netFrameworkCLRLockAndThreadsContentions.Copy(),
+		netFrameworkCLRMemoryAllocatedBytes.Copy(),
+		netFrameworkCLRMemoryFinalizationSurvivors.Copy(),
+		netFrameworkCLRMemoryHeapSize.Copy(),
+		netFrameworkCLRMemoryPromoted.Copy(),
+		netFrameworkCLRMemoryNumberGCHandles.Copy(),
+		netFrameworkCLRMemoryCollections.Copy(),
+		netFrameworkCLRMemoryInducedGC.Copy(),
+		netFrameworkCLRMemoryNumberPinnedObjects.Copy(),
+		netFrameworkCLRMemoryNumberSinkBlocksInUse.Copy(),
+		netFrameworkCLRMemoryCommitted.Copy(),
+		netFrameworkCLRMemoryReserved.Copy(),
+		netFrameworkCLRMemoryGCTime.Copy(),
+		netFrameworkCLRRemotingChannels.Copy(),
+		netFrameworkCLRContextBoundClassesLoaded.Copy(),
+		netFrameworkCLRContextBoundObjects.Copy(),
+		netFrameworkCLRContextProxies.Copy(),
+		netFrameworkCLRContexts.Copy(),
+		netFrameworkCLRRemotingCalls.Copy(),
+		netFrameworkCLRSecurityLinkTimeChecks.Copy(),
+		netFrameworkCLRSecurityChecksTime.Copy(),
+		netFrameworkCLRSecurityStackWalkDepth.Copy(),
+		netFrameworkCLRSecurityRuntimeChecks.Copy(),
+	}
+
+	// Exceptions
+	netFrameworkCLRExceptionsThrown = module.Chart{
+		OverModule: "netframework",
+		ID:         "net_framework_exception_thrown",
+		Title:      "Number of exceptions thrown since start.",
+		Units:      "exceptions",
+		Fam:        "exceptions",
+		Ctx:        "net_framework.exception_thrown",
+		Type:       module.Stacked,
+		Priority:   prioNETFrameworkCLRExceptionsThrown,
+	}
```

Comment:
I think we discussed it already. Please do either summary charts (+= every process value) or a chart per process.

Reply:
Done, PR is ready for review.